src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -62,6 +62,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_push', help="Indicates whether the image built should be pushed to the registry.", arg_type=get_three_state_flag())
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
 
+    with self.argument_context('acr image') as c:
+        c.argument('image', help='Login server with repository and tag for the source image')
+        c.argument('resource_id', help='The resource identifier of the target Azure Container Registry.')
+        c.argument('source_image', help='Repository name of the source image.')
+        c.argument('target_tags', help='List of strings of the form repo[:tag]. When tag is ommitted the source will be used.', action='append')
+        c.argument('untagged_target_repositories', help='List of strings of repository names to do a manifest only copy.', action='append')
```

Comment:
Avoid copying and reusing the strings from swagger here, they don't necessarily express the same meaning. For example, the parameter `--target_tag` can only take one `tag` (rather than a list of strings). See a few cases such as `--build-arg`, `--image`, etc.

Also are you sure about the behavior `When tag is ommitted the source will be used.`?